### PR TITLE
Add multi-app support

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-4008](https://team-1624093970686.atlassian.net/browse/AR-4008) and [AR-4009](https://team-1624093970686.atlassian.net/browse/AR-4009)

## Changes

- Added `apps.store` that maintains states of `appIds` and `appsById` https://github.com/arcana-network/developer-dashboard/pull/126/commits/7b10c6494dc7931f544a0cc506f00123002f2a3f
- Moved all types from `app.store` to `apps.store` and exported it https://github.com/arcana-network/developer-dashboard/pull/126/commits/7b10c6494dc7931f544a0cc506f00123002f2a3f
- Changed `/configure` to `/config` in routes and applied changes wherever absolute path is used https://github.com/arcana-network/developer-dashboard/pull/126/commits/5ac4c9e9fb92e32590c5f1476d7b84c97cf0b187
- Removed routing logic that are not useful (It was there for single app, not required for multiapp) https://github.com/arcana-network/developer-dashboard/pull/126/commits/091f37513925b2d73ab4457c133a32d4df49979d
- Added response type for list apps and fetch app in gateway service for better typing https://github.com/arcana-network/developer-dashboard/pull/126/commits/0786ed31932073e39592a44a21eca8cf235045b0
- Removed `app.store` from routes as it is not required for routing logic https://github.com/arcana-network/developer-dashboard/pull/126/commits/66f6191db89121b45cff26ace05c0fdde9533e20
- Created Manage Apps screen and updated styling of some components to match the new design https://github.com/arcana-network/developer-dashboard/pull/126/commits/2673b4c357e069dc1db25b487b961c440fce4f99
- Updated `createTransactionSigner` method to accept appAddress before creating the signer https://github.com/arcana-network/developer-dashboard/pull/126/commits/c4fd1f9a1d0edf1f75a113166be6b48c2927a05c
- Updated create app flow and added styling to components to match the new design https://github.com/arcana-network/developer-dashboard/pull/126/commits/c4fd1f9a1d0edf1f75a113166be6b48c2927a05c
- Extracted out delete app functionality as a separate component https://github.com/arcana-network/developer-dashboard/pull/126/commits/d30176d01b89456813159e0a31249eac3f0f7cbb
- Subscribed to `apps.store` changes in manage apps to watch for app changes https://github.com/arcana-network/developer-dashboard/pull/126/commits/86206b1c8a6b91f18fad9b4bc130d29315aa39dc
- Updated Gateway apis to accept `appId` as param https://github.com/arcana-network/developer-dashboard/pull/126/commits/a87ff075b26dfbe2f8d77e06303b05f4f2c68587 https://github.com/arcana-network/developer-dashboard/pull/126/commits/f08df2b82d7c0f66ff9bee449f920126bcafb011
- Created `Number(route.params.appId)` as composable `useAppId()` https://github.com/arcana-network/developer-dashboard/pull/126/commits/81b6a35b1c71c90f051652e8ddad4832235fe07e
- Replaced `app.store` ref with `apps.store` ref in all the components dealing with app config https://github.com/arcana-network/developer-dashboard/pull/126/commits/bdfcef091093f6131a90db65c6f1d4608514b4bd
- Removed CreateApp functionality as a page/route and made it a modal/popup in ManageApps page as per new design https://github.com/arcana-network/developer-dashboard/pull/126/commits/13aee7f051ee5c63c2fde20abed20d6354764628
- Removed `app.store` and replaced it's dependant imports with `apps.store` https://github.com/arcana-network/developer-dashboard/pull/126/commits/38c1945bb870fb3a35f563345a323ffba7479a27 https://github.com/arcana-network/developer-dashboard/pull/126/commits/b678973854574c2ec7fb4430d1dc0f4693364623
- Added `createTransactionSigner` wherever possible https://github.com/arcana-network/developer-dashboard/pull/126/commits/2c86eecd793d0c9a73b47c3c85aad4b0668f74ff https://github.com/arcana-network/developer-dashboard/pull/126/commits/ef6a3d7aef013b747c1c46f03126be3fabb37109
- Fixed spacings, paddings and other styling related things in some components

## Screencast
[Screencast from 26-08-22 12:39:59 PM IST.webm](https://user-images.githubusercontent.com/24249988/186844433-aedba240-a8e7-4094-8f07-531f62f3f402.webm)

## Screenshots
Screenshot from 2022-08-26 10-16-42
Screenshot from 2022-08-26 10-01-28
Screenshot from 2022-08-26 10-01-37

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: src/pages/ManageApps.vue [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+const canCreateApp = ref(false)
+const showDeletePopup = ref(false)
+const appToDelete = ref(0)
+
+function goToDashboard(appId: number) {
+  createTransactionSigner(appsStore.app(appId).address)
+  router.push({ name: 'Dashboard', params: { appId } })
+}
+
+function calculateLimitUsedPercent(limitUsed: number) {
```
Comment (shrinathprabhu) [inline src/pages/ManageApps.vue:44]: Rename to calculatePercentageLimitUsed

File: src/pages/ManageApps.vue [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+  return (limitUsed / MAX_ALLOWED_APP_LIMIT_IN_BYTES) * 100
+}
+
+function getProgressState(limitUsedPercent: number) {
```
Comment (shrinathprabhu) [inline src/pages/ManageApps.vue:48]: Change the order
if( <=50) success else if( > 70) error else warn

**Comment by ajithranka** [comment]
@shrinathprabhu Ready to move this out of drafts?

**Review by ajithranka — APPROVED** [review]
Looks good to merge. All comments from review calls have been addressed.

Thanks, @shrinathprabhu!

**Review by karthik-durai — APPROVED** [review]

**Pull request merged**